src/services/ono/queries/projects/index.js

```diff
@@ -0,0 +1,88 @@
+
+export const GET_TOPLEVEL_PROJECTS_QUERY = `
+query {
+	repositories {
+    nodes {
+      id
+      name
+      owner {
+        fullName
+      }
+      description
+      updated
+    }
+  }
+  groups {
+    name
+    description
+  }
+}`
+
+export const GET_PROJECTS_QUERY = `
+query($name: String!) {
+	group(name: $name) {
+    repositories {
+      nodes {
+        name
+        id
+        owner {
+          fullName
+        }
+        description
+        updated 
+      }
+    }
+    groups {
+      name
+      description 
+    }
+  }
+}`
+
+
+export type ProjectType = {
+  name: string,
+  description?: string,
```

Comment:
From the query above `description` is never undefined (that is what `description?: string` means).

But if it can be null you want to write `description: ?string`